content/widgets.md

```diff
@@ -58,6 +58,18 @@
 * Update the preferred content size
 * Wait for `viewWillTransition(to:with:)` to be called and update UI accordingly
 
+## Detecting if Device is Locked
+
+1. The first time your main app starts create a dummy file (let's call it ```ProtectionMonitor.dummy```) in a shared group container which is accessible for both the main app and the widget.
+	* You can create this file with ```NSFileManager.createFile(atPath:, contents:, attributes:)```
```

Comment:
Apple's way to denote Swift selectors is without the commas:
`NSFileManager.createFile(atPath:contents:attributes:)`

Is this a class method? Else it should probably be:
`NSFileManager.default.createFile(atPath:contents:attributes:)`, or `fileManager.createFile(atPath:contents:attributes:)` (implying that there's an instance already)